tests/test_book.py

```diff
@@ -0,0 +1,150 @@
+import io
+import json
+from unittest import TestCase, mock
+
+from src import create_app
+from src.book import create_book
+from src.constants.http_status_codes import HTTP_201_CREATED, HTTP_405_METHOD_NOT_ALLOWED,HTTP_400_BAD_REQUEST
+
+class TestBook(TestCase):
+    
+    @mock.patch("src.book.create_book", return_value=201 )
+    def test_successful_book_create(self, create_book):
```

Comment:
I don't see you mocking a call to cloudinary in this test. Your test should not actually call cloudinary.
Can you not mock an image file in Python instead? Have you tried checking that out? 🤔 
Also you can have one more test that does not have an image to be uploaded as well.